plugins/login-resources/src/index.ts

```diff
@@ -24,10 +25,14 @@ import { changePassword, getWorkspaces, leaveWorkspace, selectWorkspace, sendInv
  * Licensed under the Eclipse Public License, Version 2.0
  */
 // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+
+export { default as StatusControl } from './components/StatusControl.svelte'
```

Comment:
Why? please move it into presentation or UI if you need generic control, why include it from login-resources?

Reply:
Ok, I'll move this into UI. Thanks!